### PR TITLE
#120 Drop acronym from branch template

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,11 @@ Current issues are present in the [project backlog](https://github.com/orgs/inti
 ### Branch
 
 Each branch should be associated with a team member and corresponding issue created [here](https://github.com/intive/Picover/issues).
-To accomplish that, use following template `acronym/issue_number/description`, example:
-> kma/1/initial-project
+To accomplish that, use following template `issue_number/description`, example:
+> 1/initial-project
```
Comment (MarekMacko) [inline README.md:22]: Let me know what do you think

Comment (bartek977) [reply README.md:22]: I use alike template in my project and I can say the schema with acronym helps with finding a branch with given author.

Comment (marcinWisnia) [reply README.md:22]: on the other hand, we can see the branch author in the git history as well :D i am 60 to 40 to go with the new approach :D

Comment (MarekMacko) [reply README.md:22]: @bartek977 Agree, that can be helpful but I would say that it's really uncommon, additionally taking into consideration that usually we don't have more than 5 feature branches so it's not a big deal anyway.

**Review by marcinWisnia — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bartek977) [reply README.md:22]: @marcinWisnia I meant the acronym is helpful if you checkout on the branch using android studio plugin
image


I'm not blocking this change, for me both schemas are ok

**Review by BorysChajdas — APPROVED** [review]

**Review by michal-kucznerowicz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kmaslowiec) [reply README.md:22]: I personally like that it gives the some kind of the order in my local git:
image
But it is just my personal preference so up to you guys.

**Pull request merged**